### DISCUSSION TITLE
this is project description and i do not know where to start.

### DISCUSSION THREAD

**Discussion by vizvasrj** [ordinals/ord#2750] [General]
We are a building a BitcoinNFT marketplace for a new technology, Ordinals and BitcoinNFTs.

One of the first step is to successfully build APIs, and it requires you to have deep understanding of Bitcoin, RUST language through which Ordinals protocol is built, and should be able to run a Bitcoin node.

**Comment by raphjaph** [comment]
docs.ordinals.com
